Assets/Spawner.cs

```diff
@@ -4,12 +4,70 @@
 
 public class Spawner : MonoBehaviour
 {
+    public GameObject PlatformPrefab;
+    public GameObject FinishPrefab;
     public GameObject CoinPrefab;
-    
+    float jump_length = 4.5f; // Approximate jump length given speed, jump speed, and gravitational constant derived from expiration.
+
     void Start() {
-        for (int i = 0; i < 100; i++) {
-            var position = new Vector3(0, 0, i);
-            Instantiate(CoinPrefab, position, Quaternion.identity);
+        int platform_number = Random.Range(10, 31);
```

Comment:
This method contains many hardcoded number, it'd be better to have them adjustable in Inspector